static/templates/rt_literal_property_attrs_doc.json

```diff
@@ -100,6 +100,50 @@
       }
     ]
   },
+  {
+    "@id": "_:b6",
+    "@type": ["http://sinopia.io/vocabulary/PropertyTemplate"],
+    "http://www.w3.org/2000/01/rdf-schema#label": [
+      {
+        "@value": "Literal property attributes"
+      }
+    ],
+    "http://sinopia.io/vocabulary/hasRemark": [
+      {
+        "@value": "Whether literal property is repeatable, required, ordered, etc."
+      }
+    ],
+    "http://sinopia.io/vocabulary/hasPropertyUri": [
+      {
+        "@id": "http://sinopia.io/vocabulary/hasLiteralPropertyAttribute"
+      }
+    ],
+    "http://sinopia.io/vocabulary/hasPropertyAttribute": [
+      {
+        "@id": "http://sinopia.io/vocabulary/propertyAttribute/repeatable"
+      }
+    ],
```

Comment:
@justinlittman I'm also not sure whether this is the right value.

Reply:
This is a general attribute field; in the future in might contain attributes that are unrelated to defaults.